plugins/search/api-report.md

```diff
@@ -144,18 +150,49 @@ export const SearchContextProvider: ({
 //
 // @public (undocumented)
 export const SearchFilter: {
-  ({ component: Element, ...props }: Props): JSX.Element;
-  Checkbox(props: Omit<Props, 'component'> & Component): JSX.Element;
-  Select(props: Omit<Props, 'component'> & Component): JSX.Element;
+  ({ component: Element, ...props }: SearchFilterWrapperProps): JSX.Element;
+  Checkbox(
+    props: Omit<SearchFilterWrapperProps, 'component'> &
+      SearchFilterComponentProps,
+  ): JSX.Element;
+  Select(
+    props: Omit<SearchFilterWrapperProps, 'component'> &
+      SearchFilterComponentProps,
+  ): JSX.Element;
+  Autocomplete(props: SearchAutocompleteFilterProps): JSX.Element;
+};
+
+// @public (undocumented)
+export type SearchFilterComponentProps = {
+  className?: string;
+  name: string;
+  label?: string;
+  values?: string[];
+  asyncValues?: (partial: string) => Promise<string[]>;
+  asyncDebounce?: number;
```

Comment:
accept a Duration or suffix it with Ms perhaps, to clarify the unit?

Reply:
One thought: does this param help, or could one just do `asyncValues={debounce(fn)}` instead?

Reply:
It's a good thought; my thinking is: this should almost never _not_ be debounced, and folks will want to be able to configure it.  I ended up renaming `valuesDebounceMs` (after moving `asyncValues` -> `values` with conditional type